flink-core/src/main/java/org/apache/flink/configuration/QueryableStateOptions.java

```diff
@@ -37,10 +37,27 @@
 			key("query.server.enable")
 			.defaultValue(true);
 
-	/** Port to bind KvState server to (0 => pick random available port). */
-	public static final ConfigOption<Integer> SERVER_PORT =
+	/**
+	 * The config parameter defining the server port range of the queryable state proxy.
+	 *
+	 * <p>A proxy runs on each Task Manager, so many proxies may run on the same
+	 * machine.
+	 *
+	 * <p>Given this, and to avoid port clashes, the user can specify a port range and
+	 * the proxy is going to bind to the first free port in that range.
+	 *
+	 * <p>The specified range can be:
+	 * <ol>
+	 *     <li>a port: "9123",</li>
```

Comment:
I think the strange Javadoc convention (which is based on very old HTML) is to not have closing `li` tags